packages/venia-ui/lib/components/Header/searchTrigger.css

```diff
@@ -1,13 +1,36 @@
+:root {
+    --search-triger-width: 3rem;
+}
 .root {
     composes: root from '../clickable.css';
     transition-property: color;
     transition-duration: 224ms;
     transition-timing-function: cubic-bezier(0, 0, 0.2, 1);
     height: 3rem;
-    width: 3rem;
+    width: var(--search-triger-width);
```

Comment:
Where does this variable width come into play? Triggers (icon buttons) should all be the same square size at all resolutions.

Reply:
The desktop version button in new design has label, so I setting width to auto for wider screen size, removed global custom property and changed to property override though.